copy/observatory/faq.md

```diff
@@ -95,10 +95,60 @@ The MDN team has:
 
 ## Has the HTTP Observatory API been updated to use the new tests?
 
-Not yet. The API will continue using the old test infrastructure for a while,
+~~Not yet. The API will continue using the old test infrastructure for a while,
```

Comment:
I'm not really sure why you'd want to keep this text but in a strikethrough form.

Reply:
We don't, I left it in for a reason that escaped me ;)